README.md

```diff
@@ -1 +1,49 @@
-SFTP Inbox for LocalEGA
\ No newline at end of file
+# SFTP Inbox for LocalEGA
+
+Inbox login system
```

Comment:
don't think we should keep this line: `Inbox login system` it is fine just having the title `# SFTP Inbox for LocalEGA` - sry a bit nitpicking

Reply:
I just copied it from ReadTheDocs. But I see your point. I've updated formatting to make it a sub-header.